plugins/inputs/pgbouncer/pgbouncer.go

```diff
@@ -0,0 +1,196 @@
+package pgbouncer
+
+import (
+	"bytes"
+	"database/sql"
+	"regexp"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+
+	"github.com/lib/pq"
```

Comment:
Even if it can be performed with `postgresql_extensible`, I think they may still be an argument for having something that is ready to go.  Where to draw the line is tricky of course.

It seems to me that we should try to share as much code as possible.  Could this just be a specially initialized `postgres_extensible`?